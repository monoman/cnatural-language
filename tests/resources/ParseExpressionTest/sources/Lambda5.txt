f = p => p < x + 10 ? true : false;