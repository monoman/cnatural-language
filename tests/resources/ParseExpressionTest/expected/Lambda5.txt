Assign(Name(f) Assign Lambda(p => (Name(p) LessThan (Name(x) Add Literal(DecimalInteger, [10])))?Literal(True, [true]):Literal(False, [false]);
))